CMakeLists.txt

```diff
@@ -875,7 +875,11 @@ if(WITH_OPTIM)
         endif()
         if(WITH_CRC32_VX)
             check_vgfma_intrinsics()
-            if(HAVE_VGFMA_INTRIN)
+            if(HAVE_VGFMA_INTRIN AND
+                NOT ("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang" AND
```

Comment:
No need for quotes or ${] around CMAKE_C variables. See other examples in this cmake file.